docs/user/communication.rst

```diff
@@ -53,40 +53,42 @@ Here, users can easily query, sort, and filter existing posts.
 |course-posts|
 
 Features for Users
-------------------
+^^^^^^^^^^^^^^^^^^
 
 This section captures the most important features that are offered to any Artemis user.
 
 Search, Filter and Sort Posts
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-----------------------------
 
 On the course communication overview, a user can query *all* existing posts by different criteria.
 This can be a text-based search, context filters (i.e., a certain lecture, exercise or course-wide topic), or other post characteristics such as if it is already resolved.
 By using different sort configurations, users can find the information they are looking for at the top of the list: for example the post that got the most votes, or the post that was created most recently.
 
+A simplified version of this feature is available on the exercise and lecture details page as well.
+
 React on Posts
-^^^^^^^^^^^^^^
+--------------
 
 To foster interaction we integrate the well-known emoji reaction bar.
 Each user in the course can react on every post by making use of the emoji selection button.
-The ``+`` emoji serves as up-voting reaction, which influences the display order.
+The ``+`` emoji serves as up-voting reaction, which influences the display order of posts.
 
 Reference Posts
-^^^^^^^^^^^^^^^
+---------------
 
 If users want to refer to other posts, they can integrate a simple pattern including the hashtag (``#``) combined with the post identifier.
 A post's identifier is appended to the post title (such as seen in the screenshots above).
 
 When clicking a reference used in a post's text, the user is navigated to the referenced post.
 
-Prevent Duplicated Posts
-^^^^^^^^^^^^^^^^^^^^^^^^
+Prevent Post Duplication
+------------------------
 
 In order to prevent duplicated questions from being posted, we integrate a duplication check that runs during post creation.
 We strongly recommend users that create a post, to check the automatically provided list of similar posts to find out if the question in mind has already been asked and resolved in the best case.
```

Comment:
```suggestion
We strongly recommend users that create a post, to check the automatically provided list of similar posts to find out whether the question in mind has already been asked and resolved in the best case.
```
"whether" after infinitive verbs, instead of "if"

Otherwise no found mistakes.

Reply:
thanks!